frontend/src/app/main/ui/workspace/sidebar/options/menus/component.cljs

```diff
@@ -278,12 +308,19 @@
                          (ctv/compare-properties properties-map false)
                          (first properties-map))
 
+        malformed-map   (mapv :variant-error shapes)
+        malformed-msg   (get-variant-malformed-message malformed-map)
+
+        duplicated-ids  (->> (cfv/find-variant-components data objects variant-id)
+                             get-component-ids-with-duplicated-variant-props-and-values)
```

Comment:
Small performance improvement: 
Add here the `set` to avoid transform it in set for each component